cpuinfo.go

```diff
@@ -380,6 +380,42 @@ func parseCPUInfoMips(info []byte) ([]CPUInfo, error) {
 	return cpuinfo, nil
 }
 
+func parseCPUInfoloong64(info []byte) ([]CPUInfo, error) {
```

Comment:
IMO even if `loong32` appears later the cpuinfo format would be the same. Also please pay attention to capitalization.

```suggestion
func parseCPUInfoLoong(info []byte) ([]CPUInfo, error) {
```

Reply:
Thank you for your inspection. I have modified the code